backend/fms_core/template_importer/row_handlers/sample_submission/sample.py

```diff
@@ -189,6 +191,13 @@ def process_row_inner(self, sample, library, container, project, parent_containe
             sample['alias'] = sample['alias'] or sample['name']
             if not sample['alias']:
                 self.errors['alias'].append([f"A pooled library must have a valid alias."])
+            else:
+                # Since sample fields are not validated for pooling (through sample creation),
+                # we need to validate manually here
+                try:
+                    name_validator_without_dot(sample['alias'])
+                except ValidationError as e:
+                    self.errors['alias'] = [f"Alias {sample['alias']} is not valid."]
```

Comment:
Since most likely the input will be in the name column maybe you should classify this as "name/alias". The message could also mention the name ... "{sample['alias']} is not a valid sample name or alias."